src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  errorMessage: string;
+}
+
+export const ErrorNotification: React.FC<Props> = ({ errorMessage }) => {
+  return (
```

Comment:
you can write without return in this case
```suggestion
export const ErrorNotification: React.FC<Props> = ({ errorMessage }) => (
```